internal/fnruntime/container_test.go

```diff
@@ -19,6 +19,7 @@ package fnruntime_test
 import (
```

Comment:
@droot are these supposed to be unit tests or e2e tests?

Reply:
Actually these were added before we had `e2e` tests. I have suggested @runewake2 to implement this test in `e2e` now because with `setup.sh` support in e2e harness. We can safely remove these tests after that is done. @Shell32-Natsu Can you pl. confirm.